pkg/frontend/frontend.go

```diff
@@ -503,3 +504,37 @@ func reply(log *logrus.Entry, w http.ResponseWriter, header http.Header, b []byt
 		_, _ = w.Write([]byte{'\n'})
 	}
 }
+
+func frontendOperationResultLog(log *logrus.Entry, method string, err error) {
+	log = log.WithFields(logrus.Fields{
+		"LOGKIND":       "frontendqos",
+		"resultType":    utillog.SuccessResultType,
+		"operationType": method,
+	})
+
+	if err == nil {
+		log.Info("front end operation succeeded")
+		return
+	}
+
+	if err != nil {
```

Comment:
We don't need this check, because of the check above, we know it's not `nil`.

Reply:
Removed unneeded if statement